mars/remote/tests/sample_script.py

```diff
@@ -13,4 +13,12 @@
 # limitations under the License.
 
 import os
-assert os.environ['WORLD_SIZE'] == '2'
+
+
+def main():
+    pass
+
+
+if __name__ == "__main__":
+    assert os.environ['WORLD_SIZE'] == '2'
```

Comment:
This file seems do nothing?

Reply:
Yes, this file do nothing, I have reverted it.